pkg/render/manager.go

```diff
@@ -475,13 +488,23 @@ func (c *managerComponent) volumeMountsForProxyManager() []corev1.VolumeMount {
 
 // managerEsProxyContainer returns the ES proxy container
 func (c *managerComponent) managerEsProxyContainer() corev1.Container {
+	var keyPath, certPath string
+	if c.cfg.TLSKeyPair != nil {
+		keyPath, certPath = c.cfg.TLSKeyPair.VolumeMountKeyFilePath(), c.cfg.TLSKeyPair.VolumeMountCertificateFilePath()
```

Comment:
Shouldn't we use the trusted bundle instead of this certificate ? Isn't this created for tigera-manager service ?

Reply:
The trusted bundle is used to verify the authenticity of Linseed's certificate, but we need to mount our own certificate to present to Linseed in order to prove our own identity.